app/models/accounting/transaction.rb

```diff
@@ -22,7 +22,7 @@
 #
 # Indexes
 #
-#  acc_trans_qbid_qbtype__unq_idx                          (qb_id,qb_transaction_type) UNIQUE
+#  acc_trans_qbid_qbtype_unq_idx                           (qb_id,qb_transaction_type) UNIQUE
```

Comment:
This should be kept as `__` instead of `_`.

Reply:
I would check in with @X0nic or @linesarefuzzy on why we are using `acc_trans_qbid_qbtype__unq_idx` with double underscores. But it's not an error that there are two underscores.